documentation/CAMARA-Security-Interoperability.md

```diff
@@ -0,0 +1,188 @@
+# CAMARA Security and Interoperability Profile
+
+## Table of Contents
+<!-- TOC -->
+* Introduction
+* Audience
+* Conventions
+* General Considerations
+    * Transport Security
+* OIDC Authorization Code Flow
+    * Cross-Site Request Forgery Protection
+* Client-Initiated Backchannel Authentication Flow
+    * Authentication Request
+* Client Credentials Flow
+    * Access Token Request
+* The Scope Parameter
+* Client Authentication
+* References
+<!-- TOC -->
+
+
+## Introduction
+
+This document is the CAMARA Security and Interoperability Profile. To ensure interoperability and increased security, this technical specification restricts some options available in OIDC and CIBA, but does not change these standards.
+
+The CAMARA document sharpens the following for interoperability and security: 
+
+* Of the flows defined in OIDC CAMARA uses the [OIDC Authorization Code Flow](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth)
+
+* Scope Parameter. Recommendations about the format of the scope parameter used in  CAMARA APIs.
+
+* Client Authentication. Specifications for client authentication within CAMARA.
+
+By encapsulating these elements within this document, it aims to provide a comprehensive guide for developers and operators, ensuring consistent implementation and adherence to standardized security measures across the CAMARA ecosystem. The defined OIDC profile not only facilitates the integration process, but also serves as a basic framework for developers wishing to leverage the CAMARA APIs while maintaining security and interoperability.
+
+
+## Audience
+
+The target audience for this document is the service/technical departments of operators exposing network functions via standard CAMARA APIs. And applications or client systems that consume CAMARA standard APIs to make use of the operator's network capabilities.
+
+
+## Conventions
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+## General Considerations
+
+### Transport Security
+All network connections MUST use TLS 1.2 or better.
+
+## OIDC Authorization Code Flow
+
+The OIDC Authorization Code Flow is defined in [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html)
+
+### Cross-Site Request Forgery Protection
+
+CAMARA REQUIRES cross-site request forgery protection.
+
+CAMARA RECOMMENDS PKCE for CSRF protection. 
+CAMARA Authorization Servers MUST implement PKCE. If PKCE is not used by the Client then the CAMARA AZ must handle **state** and **nonce** as defined in OAuth2.
+
```

Comment:
As of now inoAuth2.0 PKCE is not mandatory. 
Please rephrase MUST to recommended.

Reply:
done